docs/conf.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+
+extensions = [
+    'jupyterlite_sphinx'
+]
+
+master_doc = 'index'
+source_suffix = '.md'
+
+project = 'jupyterlite-pyodide-kernel'
+copyright = 'JupyterLite Contributors'
+author = 'JupyterLite Contributors'
+
+exclude_patterns = []
+
+html_theme = "pydata_sphinx_theme"
+
+jupyterlite_dir = "."
+
+# TODO: add the pyodide logo
+# html_theme_options = {
+#    "logo": {
+#       "image_light": "TODO",
+#       "image_dark": "TODO",
+#    }
+# }
+
+def on_config_inited(*args):
+    import sys
+    import subprocess
+    from pathlib import Path
+
+    HERE = Path(__file__)
+    ROOT = HERE.parent.parent
+    subprocess.check_call([sys.executable, "-m", "pip", "install", "\".[dev]\""], cwd=str(ROOT))
```

Comment:
i've found that doing this in the `jobs` chunk of `.readthedocs.yml` helps a lot in debugging, as errors inside the RTD/sphinx output are... very hard to read.

Reply:
Indeed, it can even replace many of the needs for the conda junk with:

```
version: 2

build:
  os: ubuntu-22.04
  tools:
    python: "3.10"
    nodejs: "18"

python:
  install:
    - method: pip
      path: .
      extra_requirements:
        - docs
        - lite
```

thinking about it more: when building the docs locally, should probably _not_ force a re-install of the dev enviornment (which in turn will spawn _more_ environments, due to the build backend), and a hard js rebuild, if fixing one little typo.

Reply:
Thanks for the hint, will try it out.

Reply:
didn't want to step on your toes, so #8.